tensorflow/contrib/lite/examples/ios/simple/RunModelViewController.mm

```diff
@@ -158,9 +166,8 @@ - (NSString*)runInferenceOnImage:(UIImage *)image {
     NSString* graph_path = [self filePathForResourceName:@"mobilenet_v1_1.0_224" withExtension: @"tflite"];
```

Comment:
Get rid of all the variables above (graph, num_threads, input_layer_type and sizes that I don't think are being used? Hard to tell with github diffs :)